llmfoundry/models/mosaic_gpt/configuration_mosaic_gpt.py

```diff
@@ -51,27 +58,30 @@ def __init__(
             d_model (int): The size of the embedding dimension of the model.
             n_heads (int): The number of attention heads.
             n_layers (int): The number of layers in the model.
-            mlp_ratio (int): The ratio of the up/down scale in the MLP.
+            expantion_ratio (int): The ratio of the up/down scale in the MLP.
```

Comment:
```suggestion
            expansion_ratio (int): The ratio of the up/down scale in the MLP.
```

Reply:
Need to find-replace this throughout

Reply:
🤦‍♂️ I copy pasta'd that in a few places...
:thankyou: